<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KSecurityListEditorDecl</class>
  <widget class="QDialog" name="KSecurityListEditorDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>722</width>
        <height>453</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Security List Editor</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="K3ListView" name="m_listView">
              <column>
                <property name="text">
                  <string>ID</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Type</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Name</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Symbol</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Market</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Currency</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Fraction</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text">
                  <string>Cash Fraction</string>
                </property>
                <property name="clickable">
                  <bool>true</bool>
                </property>
                <property name="resizable">
                  <bool>true</bool>
                </property>
              </column>
            </widget>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <item>
                <spacer name="spacer6">
                  <property name="sizeHint">
                    <size>
                      <width>20</width>
                      <height>51</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="KPushButton" name="m_addButton">
                  <property name="text">
                    <string>Add...</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KPushButton" name="m_editButton">
                  <property name="text">
                    <string>Edit...</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KPushButton" name="m_deleteButton">
                  <property name="text">
                    <string>Delete...</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer3">
                  <property name="sizeHint">
                    <size>
                      <width>20</width>
                      <height>110</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QCheckBox" name="m_showCurrencyButton">
          <property name="text">
            <string>Show national currencies</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="line1">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer5">
              <property name="sizeHint">
                <size>
                  <width>91</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="KPushButton" name="m_closeButton">
              <property name="text">
                <string>Close</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global">k3listview.h</header>
    </customwidget>
  </customwidgets>
</ui>
